src/package.jl

```diff
@@ -0,0 +1,61 @@
+__precompile__()
+
+module FFTW
+
+# Since nothing is exported from AbstractFFTs as long as the FFT functionality is
+# defined (or deprecated) in Base, we need to be very explicit about the things we
+# want to import
+import AbstractFFTs: Plan, ScaledPlan,
+                     fft, ifft, bfft, fft!, ifft!, bfft!,
+                     plan_fft, plan_ifft, plan_bfft, plan_fft!, plan_ifft!, plan_bfft!,
+                     rfft, irfft, brfft, plan_rfft, plan_irfft, plan_brfft,
+                     fftshift, ifftshift,
+                     rfft_output_size, brfft_output_size,
+                     plan_inv, normalization
+
+if isdefined(Base, :FFTW)
+    import Base.FFTW: dct, idct, dct!, idct!, plan_dct, plan_idct, plan_dct!, plan_idct!
+else
+    export dct, idct, dct!, idct!, plan_dct, plan_idct, plan_dct!, plan_idct!
+end
+
+const depsfile = joinpath(dirname(@__DIR__), "deps", "deps.jl")
+if isfile(depsfile)
+    include(depsfile)
+else
+    error("FFTW is not properly installed. Please run Pkg.build(\"FFTW\") ",
+          "and restart Julia.")
+end
+
+# MKL provides its own FFTW
+fftw_vendor() = Base.BLAS.vendor() === :mkl ? :mkl : :fftw
```

Comment:
Shouldn't this function just be called `vendor()`, with a qualified name of `FFTW.vendor()` analogous to `BLAS.vendor()`?